src/menu.rs

```diff
@@ -9,12 +9,34 @@ use ratatui::{
 };
 
 pub fn view(_model: &mut Model, frame: &mut Frame) {
-    frame.render_widget(
-        Paragraph::new("Press space to start\nPress esc to Quit".to_string()),
-        frame.area(),
+    render_widget_paragraph(
+        frame,
+        "
+TERMBAN SOKOBAN 1.0 ENGINE
+==========================
+INTRODUCING... MICROBAN I!
+        BY DAVID W. SKINNER
+
+CONTROLS:       Press Enter to Begin
+                Press Escape to Leave
+                Move Player Using WASD/Arrows
+                Press R to Restart Level
+                Press Z or U to Undo a Move
+
+RULES:
+    1. The PLAYER and BOXES can only occupy EMPTY or GOAL Tiles.
+    2. The PLAYER can push a BOX onto an EMPTY Tile.
+    3. A BOX cannot be pushed by another BOX.
+GOAL:
+    The puzzle is solved when every GOAL tile is occupied by a BOX.
+",
     );
 }
 
+fn render_widget_paragraph(frame: &mut Frame<'_>, paragraph: &str) {
+    frame.render_widget(Paragraph::new(paragraph.to_string()), frame.area());
```

Comment:
It might be nice to make a box so we can style and have borders.

https://docs.rs/ratatui/latest/ratatui/widgets/block/struct.Block.html#method.inner
https://docs.rs/ratatui/latest/ratatui/widgets/block/struct.Block.html#method.style

I also used a block for the main game window so you could see how I did it there.

Reply:
save it for the sequel [(by which i mean another fine micro feature)](https://scontent.fyvr4-1.fna.fbcdn.net/v/t1.6435-9/106003200_743388533158407_922636340933383248_n.jpg?_nc_cat=103&ccb=1-7&_nc_sid=0327a3&_nc_ohc=NAKNnfQ_9EQQ7kNvgH7uegU&_nc_ht=scontent.fyvr4-1.fna&oh=00_AYBvnaujpSjvjtjkN8HDt1yC3nO5DMFDbDLVdNZNks1FtQ&oe=67013F38)